tv_shows_app/src/components/features/reviews/ReviewForm/ReviewForm.tsx

```diff
@@ -132,28 +115,17 @@ export const ReviewForm = ({ showId }: IReviewFormProps) => {
         <Text color="white" fontSize="sm">
           Rating:
         </Text>
-        <FormControl>
-          <Flex
-            flexDirection="row"
-            width={{ base: '50%', lg: '20%' }}
-            gap={1}
-            onMouseEnter={() => {
-              setLocked(false);
-            }}
-            onMouseLeave={() => {
-              setLocked(true);
-              setHoveredNumberOfStars(0);
-            }}
-            id="star-input"
-          >
+        <Controller
+          name="rating"
+          control={control}
+          render={({ field: { onChange } }) => (
             <StarReview
-              noOfStars={locked ? selectedNumberOfStars : hoveredNumberOfStars}
               isStatic={false}
-              onChange={onClick}
-              onHover={onHover}
+              onChange={onChange}
+              startNoOfStars={0}
```

Comment:
Iz Controller-a passaj value i zamijenis s ovim propom:
```suggestion
              value={value}
```